lib/build/package-analyzer.js

```diff
@@ -63,7 +63,7 @@ function determineLoaderConfig(project, description) {
         sourcePath = path.join(location, metadata.main);
       }
     } else {
-      sourcePath = path.join(location, metadata.main);
+      sourcePath = path.join(location, metadata.main || 'index.js');
```

Comment:
what if even index.js is not present? How about aborting with an error message and giving up? It generally shouldn't happen because index.js is really the kinda de-facto standard, but still you know those legacy libraries don't always play nice ;)